app/mixins/message-wall-post-notification.js

```diff
@@ -0,0 +1,29 @@
+import Mixin from '@ember/object/mixin';
+import getMessageWallOwner from '../utils/message-wall';
+
+export default Mixin.create({
+  /**
+  * Constructs a localized message wall post notification body
+  * @param {Ember.Object} model
+  * @returns {string}
+  */
+  getMessageWallPostBody(model) {
+    const firstReplierName = model.get('latestActors.0.name') || this.getTranslatedMessage('username-anonymous');
+    const wallOwner = getMessageWallOwner(model.get('url'));
```

Comment:
I think the property is `uri` not `url`

Reply:
The only other example I found for this was in `wiki-pages.js`.

Reply:
The notification model is pulled from here: https://github.com/Wikia/design-system/blob/master/app/models/wds-on-site-notifications/wds-on-site-notification.js#L33-L44
The properties you use should match up with that.